ros1_sa_build.sh

```diff
@@ -8,6 +8,8 @@ ROS_BOOTSTRAP_SCRIPT=${SCRIPT_DIR}/ros_bootstrap.sh
 "${ROS_BOOTSTRAP_SCRIPT}"
 sudo apt-get update && sudo apt-get install --no-install-recommends -y python3-colcon-common-extensions ros-${ROS_DISTRO}-ros-base
 sudo -H pip3 install -U colcon-bundle colcon-ros-bundle
+# colcon-bundle setup requirements reinstalls the latest version(50.0.0) of setuptools.
+sudo -H pip3 install -U setuptools<50.0.0 --force-reinstall
```

Comment:
Let's not put the setuptools install here, now that colcon-bundle will be updated with <50